keras/saving/pickle_utils.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Saving utilities to support Python's Pickle protocol.
+"""
+
+import tensorflow.compat.v2 as tf
+
+import os
+import tarfile
+from io import BytesIO
+from uuid import uuid4
+
+from numpy import asarray
+
+from keras.saving.save import load_model
+
+
+def unpack_model(packed_keras_model):
```

Comment:
Let's use the names `serialize_as_bytecode` and `deserialize_from_bytecode` to be more explicit.

Reply:
Sounds great, thank you for the suggestion